inverse/dipole_fit.m

```diff
@@ -291,6 +295,11 @@
   lf = lf * ori;
 end
 
+if ~isinside
```

Comment:
@arnodelorme I think that this change in https://github.com/fieldtrip/fieldtrip/pull/1139 may be relevant for you as well. Please check the whole code in the error function.